ml-agents/setup.py

```diff
@@ -30,19 +30,19 @@
     zip_safe=False,
     install_requires=[
         "mlagents_envs==0.9.3",
-        "tensorflow>=1.7,<1.8",
+        "tensorflow>=1.7,<1.15",
         "Pillow>=4.2.1",
-        "matplotlib",
-        "numpy>=1.13.3,<=1.14.5",
+        "matplotlib",  # TODO remove
+        "numpy>=1.13.3,<=1.17.0",
         "jupyter",
-        "pytest>=3.2.2,<4.0.0",
+        "pytest>=3.2.2,<4.0.0",  # TODO move to a requirements.txt file
         "docopt",
         "pyyaml",
         "protobuf>=3.6,<3.7",
-        "grpcio>=1.11.0,<1.12.0",
+        "grpcio>=1.11.0",
         "h5py>=2.9.0",
         'pypiwin32==223;platform_system=="Windows"',
     ],
-    python_requires=">=3.6,<3.7",
+    python_requires=">=3.6.1",
```

Comment:
Note that this is the only requirement that got stricter. We use some 3.6.1 features that aren't in 3.6.0 (e.g. `typing.Deque`)